docs/docs/userguide/configuration/JIRA_CONFIGURATION.md

```diff
@@ -20,6 +20,20 @@ For more information, see [Jira Version Tags](https://hub.docker.com/r/atlassian
 jira_version_tag = "<JIRA_VERSION_TAG>"
 ```
 
+### Jira image repository
+
+To change the Jira edition you can set a different image repository. By default, Jira Software edition is installed. 
+You need to make sure the appropriate version defined in `jira_version_tag` is available for the selected Jira edition.
+
```

Comment:
I think it would be nice if we mention the possible values for this variable here.